### PR TITLE
YOTJ 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Name:Albiorix, Goose Tyrant
+ManaCost:1 G U
+Types:Legendary Creature Bird
+PT:3/3
+K:Flying
+K:Trample
+K:Ward:1
+T:Mode$ Sacrificed | ValidPlayer$ You | ValidCard$ Card.token | Execute$ TrigPump | Execute$ TrigPump | TriggerDescription$ Whenever you sacrifice a token, NICKNAME perpetually gets +1/+1. This ability also triggers if NICKNAME is in exile.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt:8]: triggerzones missing

File: forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+SVar:TrigPump:DB$ Pump | PumpZone$ Battlefield,Exile | NumAtt$ 1 | NumDef$ 1 | Duration$ Perpetual
+AlternateMode:Adventure
+DeckHas:Ability$Discard|Token & Type$Food 
+DeckHints:Ability$Token & Type$Treasure|r
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt:12]: typo

File: forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Oracle:Flying, Trample, Ward {1}\nWhenever you sacrifice a token, Albiorix perpetually gets +1/+1. This ability also triggers if Albiorix is in exile.
+
+ALTERNATE
+
+Name:Wild Goose Chase
+ManaCost:U G
+Types:Instant Adventure
+A:SP$ Draw | Defined$ You | NumCards$ 2 | SubAbility$ TrigDiscard | SpellDescription$ Draw two cards, then discard two cards
+SVar:TrigDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose | SubAbility$ DBToken
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/albiorix_goose_tyrant_wild_goose_chase.txt:21]: wrong num

File: forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Emperor Apatzec Intli IV
+ManaCost:R G W
+Types:Legendary Creature Human Noble
+PT:3/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+powerGE4 | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever another creature enters the battlefield under your control, that creature perpetually gains haste if its power is 4 or greater. If its toughness is 4 or greater, you gain 4 life. If its mana value is 4 or greater, seek a creature card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt:5]: bad valid

File: forge-gui/res/editions/Alchemy Thunder Junction.txt [hunk 5]
```diff
@@ -36,3 +36,6 @@ ScryfallCode=YOTJ
 28 R Vona de ledo, the Antifex @Fajareka Setiawan
 29 U Weave the Nightmare @Wayne Wu
 30 U Runecarved Obelisk @Jorge Jacinto
+
+[token]
```
Comment (paulsnoops) [inline forge-gui/res/editions/Alchemy Thunder Junction.txt:40]: [tokens]

File: forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt (hunk 4 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt:5]: So it should be a single trigger with three conditional subabilities ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt:5]: indeed

**Review by tool4ever — APPROVED** [review]

**Pull request merged**